tardis/simulation/radiation_field_convergence.py

```diff
@@ -0,0 +1,89 @@
+from tardis.plasma.radiation_field.planck_rad_field import (
+    DilutePlanckianRadiationField,
+)
+from tardis.simulation.convergence import ConvergenceSolver
+
+
+class RadiationFieldConvergenceSolver:
+    converge_separately = True
```

Comment:
I'd like to expand this a bit on what the rate of convergence is for temperature, what rate of convergence for dilution, factor. Maybe even have individual classes for the damped and undamped.

Reply:
You would rather that be explicitly listed at the top, then modifying them with the input config, rather than just using the input config as it is now?

Reply:
I think so - similar to the plasma_solver_factory. But we can talk about this offline.